server/client.go

```diff
@@ -1226,6 +1226,17 @@ func removePassFromTrace(arg []byte) []byte {
 	return buf
 }
 
+// Returns the RTT by computing the elapsed time since now and `start`.
+// On Windows VM where I (IK) run tests, time.Since() will return 0
+// (I suspect some time granularity issues). So return at minimum 1ns.
+func computeRTT(start time.Time) time.Duration {
+	diff := time.Since(start)
+	if diff == 0 {
```

Comment:
Might say <= 0.

Reply:
Updated and replaced `diff` var name with `rtt`..